core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -2887,12 +2887,23 @@ struct ViewValueFunctor<DeviceType, ValueType, false /* is_scalar */> {
   construct_dispatch() {
     ValueType value{};
     if (Impl::is_zero_byte(value)) {
+      uint64_t kpID = 0;
+      if (Kokkos::Profiling::profileLibraryLoaded()) {
+        Kokkos::Profiling::beginParallelFor(
+            "Kokkos::View::initialization [" + name + "]",
```

Comment:
Wouldn't you want to say "memset" somewhere in the name to distinguish it?

Reply:
Sure, I am happy to do that if that finds consensus. What do you think about
```
"Kokkos::View::initialization [" + name + "] via memset", 
```
?

Reply:
I don't have strong opinions either way. Fine with having it, especially since most greps will still work

Reply:
Fine by me

Reply:
Addressed in [`4753420` (#4246)](https://github.com/kokkos/kokkos/pull/4246/commits/4753420a45f6c9a5a817e923259e5312418a5312).